sdks/go/pkg/beam/runners/prism/internal/worker/worker.go

```diff
@@ -67,7 +67,7 @@ type W struct {
 	server *grpc.Server
 
 	// These are the ID sources
-	inst, bund         uint64
+	inst               uint64
```

Comment:
This is now being pulled from the ProcessBundle instruction abstraction directly, right?

Reply:
Just the bundle descriptor IDs.

The shorter lived RPCs like Progress and Split still use the Instruction IDs from the worker side, rather than the "global" one in execute used by the ElementManager.

I haven't yet nailed down the best way to have "global" state accessed by the worker abstraction, vs dedicated state (like environment protos). It's getting there though.